src/components/sprite-info/sprite-info.css

```diff
@@ -31,49 +31,47 @@ $form-radius: calc($space / 2);
     align-items: center;
 }
 
-/* @todo: refactor from descendant selector, to a regular class */
 .icon-wrapper {
     display: inline-block;
-    box-sizing: content-box;
+
+    /* 
+        content-box is normally the browser's default. 
+        We're overriding the global, which we set to border-box 
+    */
+    box-sizing: content-box; 
     width: 1.25rem;
     height: 1.25rem;
     padding: calc($space / 2);
-    cursor: pointer;
     outline: none;
 
     /* @todo: make this a mixin for all UI text labels */
     user-select: none;
-    cursor: default;
-}
-
-.icon {
-    width: 100%;
-    height: 100%;
 }
 
-.icon.is-active { opacity: 1; }
+.radio { opacity: 0.4; cursor: pointer; }
 
-.radio-icon { 
-    opacity: 0.4;
-}
+.radio.is-active   { opacity: 1; }
+.radio.is-disabled { cursor: default; }
```

Comment:
Just rehashing what we talked about on slack: I think we should keep any CSS that is linked to a javascript action (like `cursor` <=>`onClick`) together on the same element. i.e. if the image has the onClick, then the image should have the cursor property also. 

We also talked about using more semantic markup here (button groups or actual radios) to get this kind of thing for free and also make things more tab/accessibility friendly. But that doesn't have to be part of this PR.